src/cards/dash.ts

```diff
@@ -18,11 +18,13 @@ const spell: Spell = {
     thumbnail: 'spellIconDash.png',
     description: 'spell_dash',
     effect: async (state, card, quantity, underworld, prediction) => {
+      let promises = [];
       const targets = getCurrentTargets(state);
       playDefaultSpellSFX(card, prediction);
       if (targets[0]) {
-        pull(state.casterUnit, targets[0], quantity, underworld, prediction);
+        promises.push(pull(state.casterUnit, targets[0], quantity, underworld, prediction));
```

Comment:
This could be written more simply as just `await pull(...`

No need for Promise.all(array) if there's only one.

Note: Dash used to dash to each target, in order, but this caused a significant multiplayer desync.  Because of the way forceMoves are handled on the headless server and in prediction, each spell can only cause a single `await` to occur (or else it will desync)
So a single `await Promise.all` works and a single `await promise` works but there can't be multiple `awaits` nor an `await` inside of a for loop.
When I changed Dash to fix this, I accidentally removed the await entirely which is what created the bug that you're fixing here.
But this history is important to know for future forceMove changes.  

If you're curious to learn more, see `triggerGameLoopHeadless` and `fullySimulateForceMovePredictions`, they're only invoked once after each spell effect.

Reply:
That's actually really good information. Thanks!